apps/E2E/src/Menu/specs/Menu.spec.win.ts

```diff
@@ -57,18 +61,124 @@ describe('Menu Functional Testing', () => {
+    await MenuPageObject.waitForMenuToOpen();
+
+    await MenuPageObject.clickComponent(MenuComponentSelector.FirstItem);
+    await MenuPageObject.waitForItemCallbackToFire();
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeTruthy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate MenuItem onClick() callback fired -> Press Enter', async () => {
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+
+    await MenuPageObject.sendKey(MenuComponentSelector.FirstItem, Keys.ENTER);
+    await MenuPageObject.waitForItemCallbackToFire();
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeTruthy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate MenuItem onClick() callback fired -> Press Space', async () => {
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+
+    await MenuPageObject.sendKey(MenuComponentSelector.FirstItem, Keys.SPACE);
+    await MenuPageObject.waitForItemCallbackToFire();
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeTruthy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate disabled MenuItem onClick() callback does not fire -> Click', async () => {
+    // we want to test for this because a disabled item is focusable for accessibility reasons, but it should not do anything when activated
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+
+    await MenuPageObject.clickComponent(MenuComponentSelector.SecondItem);
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeFalsy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate disabled MenuItem onClick() callback does not fire -> Press Enter', async () => {
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+
+    await MenuPageObject.sendKey(MenuComponentSelector.SecondItem, Keys.ENTER);
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeFalsy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate disabled MenuItem onClick() callback does not fire -> Press Space', async () => {
+    await MenuPageObject.clickComponent();
+    await MenuPageObject.waitForMenuToOpen();
+
+    await MenuPageObject.sendKey(MenuComponentSelector.SecondItem, Keys.SPACE);
+    await expect(await MenuPageObject.didItemCallbackFire()).toBeFalsy();
+
+    await expect(await MenuPageObject.didAssertPopup()).toBeFalsy(MenuPageObject.ERRORMESSAGE_ASSERT);
+  });
+
+  it('Validate MenuItem navigation works correctly -> Press Up + Down', async () => {
```

Comment:
Let's get a bit more specific with the test names. If these tests fail, assume the dev knows nothing about the tests.

These tests are missing the expected output.

"Validate MenuItem **keyboard** navigation works -> Press Up + Down and validate correct MenuItem has keyboard focus"